Gemfile.lock

```diff
@@ -1,57 +1,57 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actionpack (5.2.1)
-      actionview (= 5.2.1)
-      activesupport (= 5.2.1)
+    actionpack (5.2.2.1)
+      actionview (= 5.2.2.1)
+      activesupport (= 5.2.2.1)
       rack (~> 2.0)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.0.2)
-    actionview (5.2.1)
-      activesupport (= 5.2.1)
+    actionview (5.2.2.1)
+      activesupport (= 5.2.2.1)
       builder (~> 3.1)
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.0.3)
-    activemodel (5.2.1)
-      activesupport (= 5.2.1)
-    activerecord (5.2.1)
-      activemodel (= 5.2.1)
-      activesupport (= 5.2.1)
+    activemodel (5.2.2.1)
+      activesupport (= 5.2.2.1)
+    activerecord (5.2.2.1)
+      activemodel (= 5.2.2.1)
+      activesupport (= 5.2.2.1)
       arel (>= 9.0)
-    activesupport (5.2.1)
+    activesupport (5.2.2.1)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
     arel (9.0.0)
     builder (3.2.3)
-    concurrent-ruby (1.1.1)
+    concurrent-ruby (1.1.5)
     crass (1.0.4)
     double_entry (1.0.1)
       activerecord (>= 3.2.0)
       activesupport (>= 3.2.0)
       money (>= 6.0.0)
       railties (>= 3.2.0)
-    erubi (1.7.1)
-    i18n (1.1.1)
+    erubi (1.8.0)
+    i18n (1.6.0)
       concurrent-ruby (~> 1.0)
-    json (2.1.0)
-    jwt (1.5.6)
-    jwt_signed_request (2.4.0)
-      jwt (~> 1.5.0)
+    json (2.2.0)
+    jwt (2.1.0)
+    jwt_signed_request (2.5.1)
```

Comment:
### [jwt_signed_request](https://github.com/envato/jwt_signed_request)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.4.0 → 2.5.1

[_[change-log](https://github.com/envato/jwt_signed_request/blob/master/CHANGELOG.md), [source-code](https://github.com/envato/jwt_signed_request)_]





<details>
<summary>Commits</summary>

A change of **36** commits. See the full changes on [the compare page](https://github.com/envato/jwt_signed_request/compare/v2.4.0...v2.5.1).

These are the first 10 commits:
- (4bd007d) [Add project metadata to the gemspec](https://github.com/envato/jwt_signed_request/commit/4bd007dafb8d132c5eb3a95b6fb636c4afc433ef)
- (0ee3876) [Ignore the .bundle directory](https://github.com/envato/jwt_signed_request/commit/0ee38766ef5815bb32a2f8e532ba9efabe4e1b36)
- (fe18b75) [Merge pull request #30 from envato/rubygems-project-metadata](https://github.com/envato/jwt_signed_request/commit/fe18b7519caa6e537dc29f9bcd4fbcded33b8aec)
- (77fb471) [Add support for JWT version 2.1](https://github.com/envato/jwt_signed_request/commit/77fb4719f4e99f6485370a530c928527ac4a52a1)
- (b31f3ca) [Ensure bundler is installed on CI](https://github.com/envato/jwt_signed_request/commit/b31f3cab90f4661995459ae23324733649ed0cb4)
- (cbc6b1f) [Revert "Ensure bundler is installed on CI"](https://github.com/envato/jwt_signed_request/commit/cbc6b1fa3be407cd952a05c8952a0cd02f0bb7b5)
- (7db58aa) [Drop support for ruby < 2.3](https://github.com/envato/jwt_signed_request/commit/7db58aa4095727ee5b1c914c876bdd663c69ed8e)
- (0a90f14) [Merge pull request #32 from envato/travisci-bundler](https://github.com/envato/jwt_signed_request/commit/0a90f146ccdb2ed8c7d4c4f90c510da3937e3cb4)
- (3634152) [Merge pull request #28 from karnov/master](https://github.com/envato/jwt_signed_request/commit/3634152570855d704d321bb75c4a30a4f25a8b01)
- (871817c) [Bump to v2.4.1](https://github.com/envato/jwt_signed_request/commit/871817c1c3e02c9a8d3b828c2e395c43fcc88b67)


</details>